pkg/features/features.go

```diff
@@ -46,7 +46,7 @@ var (
 		DiskADController:           {Default: false, PreRelease: featuregate.Alpha},
 		DiskParallelAttach:         {Default: false, PreRelease: featuregate.Alpha},
 		DiskParallelDetach:         {Default: false, PreRelease: featuregate.Alpha},
-		DisableExpandAutoSnapshots: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
Set the featuregate to default first? code remove in next version

Reply:
please see #1220